znc/content.md

```diff
@@ -0,0 +1,33 @@
+# What is ZNC?
+
+ZNC is an IRC network bouncer (BNC). It can detach the client from the
+actual IRC server, and also from selected channels. Multiple clients
+from different locations can connect to a single ZNC account
+simultaneously and therefore appear under the same nickname on IRC.
+
+[ZNC Wiki](http://znc.in/)
+
+# How to use this image
+
+ZNC in this image stores its configuration in `/znc-data`.
+If you have existing configuration, you can reuse it with `-v $HOME/.znc:/znc-data`.
+Alternatively, you can create a new config in a volume or in a local dir.
+The examples below assumes a volume named `znc-cfg`.
+
+```console
+$ docker run -it -v znc-cfg:/znc-data znc --makeconf
+```
+
+To run ZNC:
+
+```console
+$ docker run -p 6697:6697 -v znc-cfg:/znc-data znc
+```
+
+The port should match the port you used during `--makeconf`. Note that 6667 is
+often blocked by web browsers, and therefore is not recommended.
```

Comment:
http://superuser.com/questions/188006/how-to-fix-err-unsafe-port-error-on-chrome-when-browsing-to-unsafe-ports
https://jazzy.id.au/2012/08/23/why_does_chrome_consider_some_ports_unsafe.html